code/modules/projectiles/projectile/special/temperature.dm

```diff
@@ -1,9 +1,17 @@
 /obj/item/projectile/temp
 	name = "freeze beam"
 	icon_state = "ice_2"
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
 	damage = 0
+	light_range = 2
 	damage_type = BURN
 	nodamage = FALSE
+	hitsound = 'sound/weapons/frost.ogg'
+	hitsound_wall = 'sound/weapons/frost.ogg'
+	ricochets_max = 50	//Honk!
+	ricochet_chance = 80
```

Comment:
Dunno if we really want temp 'beams' to go bouncy bounce

Reply:
this is shuttle walls